components/Articles.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import Card from './shared/Card'
+
+const Articles = () => {
+  return (
+    <>
+      <div className="text-center">
+        <h2 className="font-weight-bold">Articles</h2>
+      </div>
+      <div className="row mx-auto justify-content-center">
+        <div className="m-3">
+          <Card
+            title="IPPA Leader Series Welcomes Dr. Jenny Liu"
+            img="/images/aboutmsmr.png"
+          >
+            UPDATED: January 5, 2020 Dr. Jenny Liu delivers a presentation on
+            resilience and how it can be strengthened...
+          </Card>
+        </div>
+        <div className="m-3">
+          <Card
+            title="Resilience is More than your Mindset"
+            img="/images/maxres.jpeg"
+          >
+            Wellness Solutions podcast hosts Mary Mangos and Danny Blackford
+            explore the topic of resistance with Dr. Jenny Liu.
+          </Card>
+        </div>
+        <div className="m-3">
+          <Card
+            title="What Does it Take to Increase Our Resilience?"
+            img="/images/jumbotron.png"
+          >
+            Dr. Jenny Liu’s work is featured at the Society of Clinical
+            Psychology. A small excerpt from her article is quoted below...
+          </Card>
+        </div>
+      </div>
+    </>
+  )
+}
+
```

Comment:
since this is gonna be hydrated on build time or can you do a Object.map and default props for now?
(remember to do keys for yr react stuff or else you gonna run into issues with Object.map)

Example:
![image](https://user-images.githubusercontent.com/47992694/108578798-f6e7c000-72f1-11eb-88d7-26f6ecaf04e0.png)
![image](https://user-images.githubusercontent.com/47992694/108578803-fc450a80-72f1-11eb-900f-b293eb1fcdaa.png)



Reference:

https://github.com/Cache-and-Cookies/msmr/blob/a429c26a1fb5a2c5c28a3b4128c1c9485381c406/components/About/Acknowledgements.js#L10
https://github.com/Cache-and-Cookies/msmr/blob/a429c26a1fb5a2c5c28a3b4128c1c9485381c406/pages/about.js#L84

Reply:
so like, instead of hard-coded articles I should pass the component default props?

Reply:
yes, cuz maybe in the near future, this will be dynamic and the recent articles will change. much easier to refactor if this foresight with already in mind